flowdb/bin/build/9000_last_create_roles.sh

```diff
@@ -199,7 +193,7 @@ psql --dbname="$POSTGRES_DB" -c "
           obj record;
```

Comment:
Presumably this trigger is no longer required?

Reply:
I've kept it in to allow for the possibility of tables getting created thru routes other than flowmachine that we'd still want to be managed. That said, I'm not sure there's _really_ an eventuality where that arises and they're also in the cache table.

Reply:
I'm not sure we would necessarily want to change ownership of tables created through routes other than flowmachine, since the ownership change is specifically so that all flowmachine instances can access flowmachine-generated cache tables. Without an obvious situation where this might occur, I think it's just as plausible that we'd want the opposite - that tables created through some unforeseen route should remain owned by the user that created them.

Reply:
Yeah, I agree.